### PR TITLE
s/ECHO/ECH

### LINKED ISSUE #232: Bikeshed: finalize on naming of "ECHO"
While cutsy, the name "ECHO" for "Encrypted Client Hello" seems like it could cause problems and significant confusion.  To those not actively involved in the working group, "ECHO" has very different connotations.  Some potential risks include:
* Other IETF standards have some form of "ECHO" functionality.  When bouncing this question off someone else, they responded with "TLS ECHO, like ICMP ECHO, except totally not"
* The immediate guess people may have on hearing about a TLS ECHO functionality would be to think "oh, like to echo back a message for heartbeat purposes?".  Given the potential risk for confusion with heartbeat (especially given "Heartbleed") this has some risk of giving TLS ECHO a bad rap from the start due to confusion.
* This functionality is going to have high profile with product managers and perhaps even government regulators and public policy.  Applying a name that has confusing lay-connotations is going to muddle conversations.
* This has a risk of confusion when used in other contexts.  For example, "echoconfig" as a SVCB parameter key has potential risk for confusing people reading the config and lacking context.
* ECHO is going to be harder to do searches for, both in Internet and intranet search engines.

Just calling it "ECH" or "EHELLO" may be safer?

(If this changes, SVCB will also need to change the "echoconfig".)

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #232.

### CODE REVIEW

File: draft-ietf-tls-esni.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: Encrypted Server Name Indication for TLS 1.3
-abbrev: TLS 1.3 SNI Encryption
+title: Encrypted ClientHello for TLS 1.3
+abbrev: TLS 1.3 Encrypted ClientHello
```
Comment (tfpauly) [inline draft-ietf-tls-esni.md:3]: TLS Encrypted ClientHello —> TECH!

(Kidding, kidding)

Comment (chris-wood) [reply draft-ietf-tls-esni.md:3]: Hah!

Comment (martinthomson) [reply draft-ietf-tls-esni.md:3]: Seeing this, I think that you should remove 1.3 from the title.  Sure, this is 1.3 only, but I think that this design is fine for 1.4 also.  If it isn't, I'm sure we can deal with that eventually.  I don't want to find that we ship a 1.4 (or 4) that is compatible with this and have people tell me that it isn't compatible.

Comment (chris-wood) [reply draft-ietf-tls-esni.md:3]: That's a fair point. I'll make that change in a separate PR!

**Pull request merged**